quinn-proto/src/frame.rs

```diff
@@ -963,7 +963,7 @@ impl ObservedAddr {
         }
     }
 
-    /// Get the [`Type`] for this frame.
+    /// Get the [`FrameType`] for this frame.
     pub(crate) fn get_type(&self) -> FrameType {
```

Comment:
Should we rename this to `get_frame_type` now?

Reply:
let's just do upstream merge here?  And do any non-merge changes in a separate PR